### PR TITLE
update 8.1.0 sql script

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update SQL file with data from PR https://github.com/PrestaShop/PrestaShop/pull/27927 and https://github.com/PrestaShop/PrestaShop/pull/31130
| Type?             | improvement
| BC breaks?        |no 
| Deprecations?     | no
| Fixed ticket?     | none
| Sponsor company   | PrestaShop SA
| How to test?      | PrestaShop should install without error with this module

### CODE REVIEW

File: upgrade/sql/8.1.0.sql [hunk 1]
```diff
@@ -38,11 +36,12 @@ SELECT pa.id_product_attribute, l.id_lang, '', ''
...
+INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES
   ('PS_PRODUCT_REDIRECTION_DEFAULT', '404', NOW(), NOW()),
   ('PS_MAINTENANCE_ALLOW_ADMINS', 1, NOW(), NOW())
 ;
 UPDATE `PREFIX_product` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';
 UPDATE `PREFIX_product_shop` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';
-INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `position`) VALUES
-  (NULL, 'actionLoggerLogMessage', 'Allows to make extra action while a log is triggered', 'This hook allows to make an extra action while an exception is thrown and the logger logs it', '1');
+
+INSERT INTO `PREFIX_feature_flag` (`name`, `state`, `label_wording`, `label_domain`, `description_wording`, `description_domain`, `stability`) VALUES
```
Comment (jolelievre) [inline upgrade/sql/8.1.0.sql:46]: Careful, the feature flag actually already exists from 8.0 to 8.1
It's not that it needs to be added, it needs to be updated:
- name must be changed to `New product page`
- stability must be `stable`

In fresh install the state is also forced to 1 in 8.1, but I don't think we should force it for an upgrade.

And finally the `product_page_v2_multi_shop` feature flag must be removed from the DB

**Review by jolelievre — CHANGES_REQUESTED** [review]

File: upgrade/sql/8.1.0.sql (hunk 1 above)
Comment (jolelievre) [reply upgrade/sql/8.1.0.sql:46]: And thanks for handling this, I didn't find the time to do it myself 😉

Comment (matthieu-rolland) [reply upgrade/sql/8.1.0.sql:46]: oh nice catch 👍

Comment (matthieu-rolland) [reply upgrade/sql/8.1.0.sql:46]: done 👍

**Review by Hlavtox — APPROVED** [review]

**Review by HanaRebaiQA — APPROVED** [review]
Hello @matthieu-rolland 

Thanks for your PR! I have checked it and here is my feedback : 

-Upgrade from PS1788 to PS8.0.1 : OK
-Rollback from PS8.0.1 to PS1788 : OK
-Upgrade from PS1788 to PS8.1.0 : OK
-Rollback from PS8.1.0 to PS1788 : OK
-Upgrade from PS8.0.1 to PS8.1.0 : OK
-Rollback from PS8.1.0 to PS8.0.1 : OK
-Check with multistore : OK
- Check the product page V2 : OK
- Check the Feature flag for the product V2 : OK

image

image

image

So, it's QA :heavy_check_mark: 

Thank you!

**Comment by nicosomb** [comment]
Thank you @matthieu-rolland !

**Pull request merged**